fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -484,6 +484,7 @@ class TestConfig : public BasicTestConfig {
 	// Refer to FDBTypes.h::TLogVersion. Defaults to the maximum supported version.
 	int maxTLogVersion = TLogVersion::MAX_SUPPORTED;
 	int extraMachineCountDC = 0;
+	int extraStorageMachineCountDC = 0;
```

Comment:
is this per DC? if so, maybe use `extraStorageMachineCountPerDC`

Reply:
Seems that DC suffix indicates the PerDC? following the same naming as the extraMachineCountDC

Reply:
Change both? PerDC does seem more clear but not sure if there are other instances (other files) where we use DC suffix instead of PerDC.